src/scheduler_utils.h

```diff
@@ -28,10 +28,17 @@
 #include <deque>
 #include <unordered_map>
 
+#include "cache_manager.h"
 #include "scheduler.h"
 
 namespace triton { namespace core {
 
+uint64_t CaptureTimeNs();
+bool CacheLookUpUtil(
```

Comment:
May I ask to provide a description of what this util is doing above its declaration?

Reply:
Added description.

Reply:
May I ask to follow the standard we are using in the code, please. Here's the example: https://github.com/triton-inference-server/core/blob/main/src/backend_config.h

Reply:
Is this function is intended to be used by dynamic batch scheduler and ensemble scheduler only ?

Reply:
or potentially can be universal?

Reply:
As of now, it's being caled from these two schedulers only. I think up until this change, it's only called by dynamic batch scheduler.
But if we add more schedulers or other schedulers to support response caching, then it can be used those schedulers too.